subworkflows/local/arg.nf

```diff
@@ -85,10 +85,21 @@ workflow ARG {
     // RGI run
     if ( !params.arg_skip_rgi ) {
 
-        // Download and prepare CARD
-        UNTAR ( [ [], file('https://card.mcmaster.ca/latest/data', checkIfExists: true).copyTo(params.outdir + '/databases/card/data.tar.gz') ] )
-        ch_versions = ch_versions.mix( UNTAR.out.versions )
-        RGI_CARDANNOTATION ( UNTAR.out.untar.map{ it[1] } )
+        if ( !params.arg_rgi_database ) {
+
+            // Download und untar CARD
+            UNTAR ( [ [], file('https://card.mcmaster.ca/latest/data', checkIfExists: true).copyTo("${params.outdir}/databases/rgi/data.tar.gz") ] )
```

Comment:
As the with the other I would just publish the UNTAR results rather than `copyTo`